packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
```

Comment:
Please change this to `eslint-config-godaddy`. From a combinatoric perspective there are a couple of ways to approach this:

1. Disable these rules in `eslint-config-godaddy` and then have another config which is these rules turned on. 
2. Have a `minimal.js` inside each of the other packages that disables these rules, but includes all other rules. 
3. Another approach I haven't thought of.